netlify.toml

```diff
@@ -1,3 +1,26 @@
-[[plugins]]
-  package = "@netlify/plugin-nextjs"
+# netlify.toml for Next.js static export
+[build]
+  command = "npm run build"
+  publish = "out"
 
+# Redirect all traffic to root HTML so that next/router can do its thing
+[[redirects]]
+  from = "/*"
+  to = "/index.html"
+  status = 200
```

Comment:
This should theoretically fix our routing issue on its own. The idea is that this forces static site behavior, where we. have one HTML document (`index.html`) that has a chonky javascript file that simulates all the routing and rendering. Unfortunately, it seems that Next v14+ introduced a bug where the router does not initialize correct for dynamic routes. Discussion [here](https://github.com/vercel/next.js/discussions/55393). This is the problem we're dealing with 😭